### PR TITLE
Drive straight - with changes

### PR DESCRIPTION
**Pull request by bczdog**

### CODE REVIEW

File: 2022-RapidReact/src/main/java/frc/robot/commands/DriveStraigt.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+// BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=IMPORTS
+import frc.robot.subsystems.Drivetrain;
+
+    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=IMPORTS
+
+/**
+ *
+ */
+public class DriveStraigt extends CommandBase {
```
Comment (sanjana36) [inline 2022-RapidReact/src/main/java/frc/robot/commands/DriveStraigt.java]: Please update the spelling of DriveStraight

**Review by sanjana36 — CHANGES_REQUESTED** [review]
please update DriveStraight spelling

**Comment by AdriaanFriesen** [comment]
Fixed name, reopening

**Comment by bczdog** [comment]
Fixes confirmed

**Comment by bczdog** [comment]
No longer required.

**Pull request closed without merging**